scripts/release/build/print-post-build-summary.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const chalk = require('chalk');
+const {getUnexecutedCommands} = require('../utils');
+
+const CHANGELOG_PATH =
+  'https://github.com/facebook/react/edit/master/CHANGELOG.md';
+
+module.exports = params => {
+  const command =
+    `./publish.js -v ${params.version}` +
+    (params.path ? ` -p ${params.path}` : '') +
+    (params.dry ? ' --dry' : '');
+
+  console.log(
+    chalk`
+    {green.bold Build successful!}
+    ${getUnexecutedCommands()}
+    Next there are a couple of manual steps:
+
+    {bold.underline Step 1: Update the CHANGELOG}
+
+    Here are a few things to keep in mind:
+    • The changes should be easy to understand. (Friendly one-liners are better than PR titles.)
+    • Make sure all contributors are credited.
+    • Verify that the markup is valid by previewing it in the editor: {blue.bold ${CHANGELOG_PATH}}
+
+    {bold.underline Step 2: Smoke test the packages}
```

Comment:
Could this be automated using WebDriver?

Reply:
_Maaaybe_ but I am not a very big fan of WebDriver automated tests. 😄

Reply:
I think @nhunzaker is looking at this.

Reply:
We can always follow up with that. I'm reluctant about introducing WebDriver into our release process. It's always a pain in the ass. But I won't be stubborn about it if others feel differently.

Reply:
I agree. I would hate for a webdriver test to block a release. I think we'd be more successful sending out edge releases so that issue filers can quickly verify their fixes.

I hadn't thought of using webdriver for the packages, only specifically for the manual DOM test fixtures.

Reply:
Good news is, the new release script should make preview releases easy too!

```sh
./scripts/release/build.js -v 16.1.0-rc.0
# fixture verification...
./scripts/release/publish.js -v 16.1.0-rc.0
```

Reply:
I also like the idea of automated tests for the fixtures stuff- but I would prefer not to put it into the release script. The release script could make mention of the automated scripts though, just not block on them.